app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CategoriesController < ApplicationController
+
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params)
+    if @category.save
+      redirect_to category_path(@category)
+    else
+      render :new
```

Comment:
Don't forget to handle the errors. What happens if a user enters something incorrectly and the Category doesn't get created? The user should see a nice error message.